os/other-settings.md

```diff
@@ -82,6 +82,25 @@ Further details can be found in the systemd man pages:
 
 The Container Linux bootloader parses the configuration file `/usr/share/oem/grub.cfg`, where custom kernel boot options may be set.
 
+The `/usr/share/oem/grub.cfg` file can be configured with ignition. Note that ignition runs after grub. Therefore, the grub configuration won't take effect until the next reboot of the node. 
+
+```yaml container-linux-config
+storage:
+  filesystems:
+    - name: "OEM"
+      mount:
+        device: "/dev/disk/by-label/OEM"
+        format: "ext4"
+  files:
+    - filesystem: "OEM"
+      path: "/grub.cfg"
+      mode: 0644
+      append: true
+      contents:
+        inline: |
+          set linux_append="coreos.autologin=tty1"
```

Comment:
This should be `set linux_append="$linux_append coreos.autologin=tty1"`.  Some OEMs already set that variable.